guacamole/src/main/java/org/apache/guacamole/rest/auth/HashTokenSessionMap.java

```diff
@@ -226,6 +226,7 @@ public GuacamoleSession remove(String authToken) {
     @Override
     public void shutdown() {
         executor.shutdownNow();
+        sessionMap.values().stream().forEach(GuacamoleSession::invalidate);
```

Comment:
It is somewhat obvious what's happening here, but a comment explaining it could maybe still be nice, especially since it's new (and not entirely clear to me why it's needed now, but not before)

Reply:
Some of that should be captured by the commit message of the commit that introduced this line (affdcc8e425786ddd99a1375a93fce7c0a1307ef):

>  GUACAMOLE-1224: Ensure all user sessions are explicitly invalidated prior to application shutdown.

I can add a similar comment to clarify the code here.